src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -42,3 +42,14 @@ target_link_libraries(
   PRIVATE
   GeneralRelativitySolutions
   )
+
+add_generalized_harmonic_executable(
+  KerrSchildNumericInitialData
+  evolution::NumericInitialData<GeneralizedHarmonic::System<3>>
+  GeneralizedHarmonic::Solutions::WrappedGr<gr::Solutions::KerrSchild>
+)
+target_link_libraries(
+  EvolveKerrSchildNumericInitialData
+  PRIVATE
+  GeneralRelativitySolutions
```

Comment:
Since the analytic and numeric executables both do link against GeneralRelativitySolutions, why split it off?

Reply:
I figured since I'm changing the cmake file I might as well do that now. Once we have other boundary conditions the numeric initial data exec won't need to link the solutions anymore.